pokeapp/src/components/Sidebar.js

```diff
@@ -69,117 +71,119 @@ const SideBar = ({
-          </div>
-        </div>
+          <div className="type-weakness">
+            <div className="sidebar-type">
+              <h4>Type</h4>
+              <div>{typeIconMapper(selectedPokemon.types)}</div>
+            </div>
 
-        <div className="btn-container">
-          <div className="buy-sell-container">
-            <button
-              className="buy-btn"
-              onClick={() => {
-                dispatch(buyPokemon(selectedPokemon));
-              }}
-            >
-              <h4>Buy</h4>
-            </button>
-            <button
-              className="sell-btn"
-              onClick={() => {
-                dispatch(sellPokemon(selectedPokemon));
-              }}
-            >
-              <h4>Sell</h4>
-            </button>
+            <div className="sidebar-weakness">
+              <h4>Weakness</h4>
+              <div>{weaknessIconMapper(selectedPokemon.weaknesses)}</div>
+            </div>
           </div>
 
-          <div className="more-info-btn">
-            {!isMoreInfo ? (
-              <h4
+          <div className="btn-container">
+            <div className="buy-sell-container">
+              <button
+                className="buy-btn"
                 onClick={() => {
-                  setIsMoreInfo(true);
+                  dispatch(buyPokemon(selectedPokemon));
                 }}
               >
-                More Information
-              </h4>
-            ) : (
-              <h4
+                <h4>Buy</h4>
+              </button>
+              <button
+                className="sell-btn"
                 onClick={() => {
-                  setIsMoreInfo(false);
+                  dispatch(sellPokemon(selectedPokemon));
                 }}
               >
-                Less Information
-              </h4>
-            )}
-          </div>
+                <h4>Sell</h4>
+              </button>
+            </div>
 
-          <div>
-            {isMoreInfo ? abilityMapper(selectedPokemon.abilities) : null}
-          </div>
-          <div>
-            {isMoreInfo ? gameMapper(selectedPokemon.games) : null}
-            {!selectedPokemon.games.length && isMoreInfo ? (
-              <p>This Pokemon has not been in any games</p>
-            ) : null}
-          </div>
-          <div>
-            {isMoreInfo ? <p>{selectedPokemon.height}</p> : null}
-            {isMoreInfo ? <p>{selectedPokemon.weight}</p> : null}
+            <div className="more-info-btn">
+              {!isMoreInfo ? (
+                <h4
+                  onClick={() => {
+                    setIsMoreInfo(true);
+                  }}
+                >
+                  More Information
+                </h4>
+              ) : (
+                <h4
+                  onClick={() => {
+                    setIsMoreInfo(false);
+                  }}
+                >
+                  Less Information
+                </h4>
+              )}
+            </div>
```

Comment:
It might be better to have the div manage the click event so the user doesn't have to click directly on the text to get it to work.

Reply:
Great idea. I will make this addition in a new PR.